### PR TITLE
Add yyyy-mm-01 as a new Invoice numbering scheme

### PR DESCRIPTION
**Pull request by viadanna**
*JIRA ticket*: [OC-4667](https://tasks.opencraft.com/browse/OC-4667)

**Testing instructions**:

1. Start devstack and access the admin. (I recommend Vagrant)
2. Create new *Template* and select `yyyy-mm-01` as numbering scheme.
3. Create an **Invoice**  using the previously created template.
4. Use the shell to run `utils.send_email_with_invoice` that generates an invoice with the correct template.

**Reviewers**
- @itsjeyd

### CODE REVIEW

File: accounting/invoice/choices.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ class InvoiceNumberingScheme(DjangoChoices):
     year_month_number = ChoiceItem('%Y-%m-{number}', _('yyyy-mm-{number} (2018-01-42, for the 42nd invoice)'))
     opencraft_year_month = ChoiceItem('OC-%Y-%m', _('OC-yyyy-mm (OC-2018-01)'))
     opencraft_number = ChoiceItem('OC-{number}', _('OC-{number} (OC-42, for the 42nd invoice)'))
+    year_month_one = ChoiceItem('%Y-%m-01', _('yyyy-mm-01 (2018-01-01, as just one is expected)'))
```
Comment (itsjeyd) [inline accounting/invoice/choices.py]: @viadanna Could you move this option up so it sits below `year_month_number`? It's more closely related to that format than the `opencraft_*` numbering schemes.

File: accounting/invoice/choices.py [hunk 2]
```diff
@@ -72,9 +72,9 @@ class InvoiceNumberingScheme(DjangoChoices):
     default = ChoiceItem('%Y-%m', _('yyyy-mm (2018-01)'))
     number = ChoiceItem('{number}', _('{number} (42, for the 42nd invoice)'))
     year_month_number = ChoiceItem('%Y-%m-{number}', _('yyyy-mm-{number} (2018-01-42, for the 42nd invoice)'))
+    year_month_one = ChoiceItem('%Y-%m-01', _('yyyy-mm-01 (2018-01-01, 2018-02-01 etc.)'))
```
[the remark this answers is not in the document]
Comment (viadanna) [reply accounting/invoice/choices.py]: Oh, alright.

**Comment by itsjeyd** [comment]
@viadanna Dang, I was just about to comment about squashing commits before merging! Could you do that next time? :) It's not necessary to always squash things down to a *single* commit, but commits that just make small adjustments to previous commits that implement a given feature (such as ce1f225294fc65b9ac3f611dd18e316fc38d40e3 and 6de909a6c4c4316d717092269364df89f384b021) should be squashed into the appropriate commits before merging.

**Comment by itsjeyd** [comment]
@viadanna Ah, great! I was only looking at this PR and didn't check the list of commits in master 😊

**Pull request merged**